kbase-extension/static/kbase/js/widgets/narrative_core/kbaseModal.js

```diff
@@ -0,0 +1,136 @@
+/**
```

Comment:
Another modal widget? This might be okay for now, but we should agree on one that serves all needs.